CMakeLists.txt

```diff
@@ -289,8 +289,7 @@ if(BUILD_COMMON_LWS)
       set_target_properties(kvsCommonLws PROPERTIES VERSION ${KINESIS_VIDEO_PRODUCER_C_VERSION} SOVERSION ${KINESIS_VIDEO_PRODUCER_C_MAJOR_VERSION})
     endif()
     target_link_libraries(kvsCommonLws
-            ${OPENSSL_CRYPTO_LIBRARY}
-            ${OPENSSL_SSL_LIBRARY}
+            ${PRODUCER_CRYPTO_LIBRARY}
```

Comment:
Shouldn't there be an if-else for openssl and mbedtls and ${OPENSSL_SSL_LIBRARY} removed only for mbedtls?

Reply:
https://github.com/awslabs/amazon-kinesis-video-streams-producer-c/pull/442/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR263-R270